src/client/clientmap.h

```diff
@@ -87,10 +86,18 @@ class ClientMap : public Map, public scene::ISceneNode
 
 	void updateCamera(const v3f &pos, const v3f &dir, f32 fov, const v3s16 &offset)
 	{
+		v3s16 previous_block = getContainerPos(floatToInt(m_camera_position + intToFloat(m_camera_offset, BS), BS), MAP_BLOCKSIZE);
```

Comment:
isn't this equivalent?
```suggestion
		v3s16 previous_block = getContainerPos(floatToInt(m_camera_position, BS) + m_camera_offset, MAP_BLOCKSIZE);
```